constants.py

```diff
@@ -3,21 +3,43 @@
 
 class Dataset(Enum):
     QM3_1 = 'QM3.1'
-    PPD1_1 = 'PPD1.1'
-    PPD1_2 = 'PPD1.2'
-    PPD1_3 = 'PPD1.3'
     PPD1_6 = 'PPD1.6'
-    PPD1_7 = 'PPD1.7'
-    PPD1_8 = 'PPD1.8'
 
 
 class PrintTemplate:
     QM_PRINT_FILE_TEMPLATE = (
-        'UAC', 'QUESTIONNAIRE_ID', 'WALES_UAC', 'WALES_QUESTIONNAIRE_ID', 'TITLE', 'COORDINATOR_ID', 'FORENAME',
+        'UAC',
+        'QUESTIONNAIRE_ID',
+        'WALES_UAC',
+        'WALES_QUESTIONNAIRE_ID',
+        'TITLE',
+        'COORDINATOR_ID',
```

Comment:
Not something you've done but I think these need to be swapped to match the spec